app/controllers/deploys_controller.rb

```diff
@@ -184,7 +184,7 @@ def search
       deploys = deploys.where("updated_at between ? AND ?", *updated_at)
     end
     deploys = deploys.where.not(deleted_at: nil) if search_deleted
-    pagy(deploys, page: page, items: 30)
+    pagy(deploys, page: params[:page], items: 30)
```

Comment:
do you need to permit :page or are you not using strong params?

Reply:
no, params are freely useable unless they are assigned to models